youtube_dl/extractor/nrk.py

```diff
@@ -210,7 +210,14 @@ def _get_subtitles(self, subtitlesurl, video_id, baseurl):
             duration = parse_duration(p.get('dur'))
             starttime = self._subtitles_timecode(begin)
             endtime = self._subtitles_timecode(begin + duration)
-            srt += '%s\r\n%s --> %s\r\n%s\r\n\r\n' % (compat_str(pos), starttime, endtime, p.text)
+
+            subs = []
+            subs.append(unicode(p.text).strip())
```

Comment:
`unicode` is not defined in python 3.x, you'll have to use `compat_str`. But are you sure it's needed? I think `p.text`  is always an unicode string.